src/pages/RequestView/RequestView.test.tsx

```diff
@@ -66,10 +67,10 @@ describe('RequestView', () => {
     expect(screen.getByText('SUCCESS')).toBeInTheDocument()
 
     expect(screen.getByText('Created')).toBeInTheDocument()
-    expect(screen.getByText('Nov 3, 2022, 23:12:46')).toBeInTheDocument()
```

Comment:
This is the correct time in UTC. What has changed such that the time is actually shown on the page in local time is unknown.